docs/pages/deployment/configuration.rst

```diff
@@ -85,4 +85,6 @@ As a general safety precaution ``auth.contractvalidators`` ignores the ``dummy``
 requesting an access token from another node on ``/n2n/auth/v1/accesstoken`` does not return any error details,
 ``http.default.cors.origin`` does not allow a wildcard (``*``),
 json-ld context can only be downloaded from trusted domains configured in ``jsonld.contexts.remoteallowlist``,
-and the ``internalratelimiter`` is always on.
\ No newline at end of file
+and the ``internalratelimiter`` is always on.
+
+Interacting with remote Nuts nodes requires HTTPS: it will refuse to connect to plain HTTP endpoints when in strict mode.
```

Comment:
this is very vague. "internal" endpoints (/internal, /health, /status, ...) do not require https

Reply:
aah, missed the `remote` part